README.md

```diff
@@ -38,7 +38,9 @@ The configuration file follows a simple structure.
             "name": "A name for the query",
             "description": "Description of the query",
             "id": "A unique ID for the query",
-            "sources": "Sources over which the query should be executed"
+            "sources": "Sources over which the query should be executed",
+            "trueText": "The text that is to be shown when the query result is true, only useful for ASK queries.",
```

Comment:
In this case I would add an object like:
```json
"askQuery": {
  "trueText": "...",
  "falseText": ",,,"
}
```

Reply:
I've made a seperate object for this in [commit 3af2e39](https://github.com/SolidLabResearch/generic-data-viewer/pull/42/commits/3af2e3971f964d6f0d92a2690bd045c48e6cd079)